2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 4 Line 5_
```markdown
## Strings

Strings are a very important data type in all languages. In Python, strings may be quoted several ways:

### Construction
```
PEP8 style is to use double quotes.

I'm happy to run the modules through a PEP8 linter at the end to catch any stragglers.

And I want to spend zero energy during the course enforcing PEP8 on the students!

But for future modules, keeping our examples as close to PEP8 as possible will make that formatting work easier down the road.

Reply:
added a lil blurb recommending double quotes! I agree PEP 8 is the way to go.